app/src/main/java/com/android/voyageur/ui/overview/Overview.kt

```diff
@@ -112,6 +127,8 @@ fun OverviewScreen(
             userViewModel)
       },
       content = { pd ->
+        // if the device is not conencted to internet show a text
```

Comment:
Here I see you added this comment, but you don't show a text here? I see you show one above, is this what you meant?